certified-connectors/BttnV2/apiDefinition.swagger.json

```diff
@@ -0,0 +1,284 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "BttnV2",
+    "description": "The magical bttn that automates your business, improves your customer experience, and so much more\u2026",
+    "version": "1.0",
+    "contact": {
+      "name": "Bttn support",
+      "url": "https://bttn.freshdesk.com/support/solutions",
+      "email": "support@bt.tn"
+    }
+  },
+  "host": "powerautomate.bt.tn",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/company/1": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@WesselBruijnes Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
Applies for all the responses in the swagger